docs/src/dev/future/index.asciidoc

```diff
@@ -49,7 +49,6 @@ line of 22H2.
 
 * Add support for traversals as parameters for `V()`, `is()`, and `has()` (includes `Traversal` arguments to `P`)
 * Geospatial support for TinkerPop (link:++https://lists.apache.org/list?dev@tinkerpop.apache.org:2021-7:DISCUSS%20geo-spatial++[DISCUSS Thread])
-* Add mid-traversal `E()` support
```

Comment:
could you please create a JIRA for this and reference it here rather than deleting the line? if folks want to know the status they could just look in on the ticket. i believe that's how it was being done prior to the 3.6.x release. it would be good to get the JIRA on this PR too so that it all links together nicely.

Reply:
Added ticket https://issues.apache.org/jira/browse/TINKERPOP-2798

Reply:
added in https://github.com/apache/tinkerpop/pull/1808/commits/13501c50233d1692d2cd3dd8185e61e9050f8fe3